### PR TITLE
Jgkamat's First Goodie!

### PR DESCRIPTION
**Pull request by jgkamat**
# Goodie Pull Request Template

Using this template will help us better understand your Instant Answer and assist you when necessary.  Simply copy and paste the markdown below into the description of your GitHub pull request and complete as appropriate.

**What does your Instant Answer do?**
Tells the world 'jgkamat' is awesome!

**What problem does your Instant Answer solve (Why is it better than organic links)?**
Teaching me DDH

**What is the data source for your Instant Answer? (Provide a link if possible)**
A string

**Why did you choose this data source?**
N/A

**Are there any other alternative (better) data sources?**
N/A

**What are some example queries that trigger this Instant Answer?**
duckduckhack jgkamat

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
N/A

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
Yes?

**Which existing Instant Answers will this one supercede/overlap with?**
N/A

**Are you having any problems? Do you need our help with anything?**
Nope, let me know if I messed up anywhere though!

**Where did you hear about DuckDuckHack? (For first time contributors)**
The Gatech Linux User Group!

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
snapshot76

## Checklist
Please place an 'X' where appropriate.

```
- [x] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
- [x] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
- [x] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
- [x] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
- [x] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Linux 3.16
         - [x] Google Chrome
         - [x] Firefox
         - [x] Opera

    - Windows 8
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 10

    - Windows 7
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 8
        - [] IE 9
        - [] IE 10

    - Windows XP
        - [] IE 7
        - [] IE 8

    - Mac OSX
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] Safari

    - iOS (iPhone)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - iOS (iPad)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - Android
        - [] Firefox
        - [] Native Browser
        - [] Google Chrome
        - [] Opera

```

### CODE REVIEW

**Comment by moollaza** [comment]
LGTM :+1:

**Comment by moollaza** [comment]
@jgkamat thanks a lot for contributing (we really appreciate it :smiley:), and congrats on making your first Goodie!

Now that you've made your first Goodie, we really hope you'll continue to contribute and make more Instant Answers! Please checkout our [Ideas forum](https://duck.co/ideas) if you're unsure about what to build. There's also lots of Github Issues in our repos that need someone to tackle them!

File: lib/DDG/Goodie/IsAwesome/jgkamat.pm [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+attribution github => ["https://github.com/jgkamat", "jgkamat"],
+            twitter => "onion_chesse";
+
+# Triggers
+triggers start => "duckduckhack jgkamat";
+
+# Handle statement
+handle remainder => sub {
+    return if $_;
+	return "jgkamat is awesome and has successfully completed the DuckDuckHack Goodie tutorial!";
```
Comment (mintsoft) [inline lib/DDG/Goodie/IsAwesome/jgkamat.pm]: Can you fix the indentation here? It should be 4 spaces :)

**Comment by jgkamat** [comment]
oops, that was my bad, I've just fixed it... 
I dont think I see any other indentation errors :P

**Comment by mintsoft** [comment]
Thanks for the quick fix @jgkamat Congratulations on your first goodie!

**Comment by moollaza** [comment]
@jgkamat congrats! It's live: https://duckduckgo.com/?q=duckduckhack+jgkamat&ia=answer

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's & Issues or even submit more Instant Answers!

**Comment by jgkamat** [comment]
Thanks! I'll definetly stick around, and make some more goodies/instant answers. Thanks for being such a great community! :smile:

**Pull request merged**